integration-tests/integration/roles.spec.js

```diff
@@ -0,0 +1,89 @@
+const RolesPage = require('../pages/rolesPage')
+const MenuPage = require('../pages/menuPage')
+const { replicateRoles } = require('../support/roles.helpers')
+
+context('Roles', () => {
+  before(() => {
+    cy.clearCookies()
+  })
+
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('Should display all roles', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_OAUTH_USERS' }, { roleCode: 'ROLES_ADMIN' }] })
+    cy.signIn()
+
+    cy.task('stubAllRoles', {})
+    MenuPage.verifyOnPage().manageRoles()
+
+    const roles = RolesPage.verifyOnPage()
+
+    roles.rows().should('have.length', 3)
+    roles.rows().eq(0).should('contain.text', 'Global Search')
+    roles.rows().eq(1).should('contain.text', 'Licence Responsible Officer')
+    roles.rows().eq(2).should('contain.text', 'Licence Vary')
+  })
+
+  it('Should display paged results for all roles', () => {
+    cy.task('stubSignIn', { roles: [{ roleCode: 'MAINTAIN_OAUTH_USERS' }, { roleCode: 'ROLES_ADMIN' }] })
+    cy.signIn()
+    cy.task('stubAllRoles', {
+      content: replicateRoles(5),
+      totalElements: 21,
+      page: 0,
+      size: 5,
+    })
+    MenuPage.verifyOnPage().manageRoles()
+
+    const roles = RolesPage.verifyOnPage()
+
+    roles.rows().should('have.length', 5)
+    // roles.rows().eq(0).should('include.text', 'Role\u00a0Name\u00a00')
+    // roles.rows().eq(1).should('include.text', 'Role\u00a0Name\u00a01')
+    // roles.rows().eq(2).should('include.text', 'Role\u00a0Name\u00a02')
+    // roles.rows().eq(3).should('include.text', 'Role\u00a0Name\u00a03')
+    // roles.rows().eq(4).should('include.text', 'Role\u00a0Name\u00a04')
```

Comment:
Better to remove the commented out code?

Reply:
will remove on next PR